src/plugins/finalcutpro/inspector/video.lua

```diff
@@ -144,25 +145,43 @@ end
 local function doStabilizationMethod(value)
     local timeline = fcp:timeline()
     local timelineContents = timeline:contents()
+    local stabilization = fcp:inspector():video():stabilization()
     local method = fcp:inspector():video():stabilization():method()
 
-    return Do(function()
+    return If(function()
         --------------------------------------------------------------------------------
         -- Make sure at least one clip is selected:
         --------------------------------------------------------------------------------
         local clips = timelineContents:selectedClipsUI()
         if clips and #clips == 0 then
-            log.ef("Set Blend Mode Failed: No clips selected.")
-            tools.playErrorSound()
+            displayMessage(i18n("noSelectedClipsInTimeline"))
             return false
+        else
+            return true
         end
-
-        return Do(method:doSelectValue(value))
-        :Then(WaitUntil(method):Is(value):TimeoutAfter(2000))
-        :Then(true)
-    end)
+    end):Is(true):Then(
+        If(stabilization:doShow()):Is(true)
```

Comment:
The `If` will default to checking if it `Is(true)`, so that can be skipped. i.e.
```suggestion
        If(stabilization:doShow())
```